content/en/blog/_posts/2021-03-09-Defining-NetworkPolicy-Conformance-For-CNIs

```diff
@@ -0,0 +1,373 @@
+---
+layout: blog
+title: "Defining NetworkPolicy Conformance for CNI providers"
+date: 2021-03-21
+slug: defining-networkpolicy-conformance-cni-providers
+---
+
+Authors: Matt Fennwick, Jay Vyas, Ricardo Katz, Amim Knabben, Douglas Landgraf
+
+A brief conversation around "node local" network policies in April of 2020 inspired the creation of a NetworkPolicy subproject from sig-network.
+
+In this post we'll discuss:
+
+- Why we created a Subproject for Network Policies 
```

Comment:
Maybe link to https://kubernetes.io/docs/concepts/services-networking/network-policies/ ?

Reply:
sure